src/apns.app.src

```diff
@@ -13,6 +13,9 @@
   ]},
   {modules, []},
   {mod, {apns_app, []}},
+  {env, [
+      {openssl_command, "openssl"}
```

Comment:
we should write some doc about this in the README explaining it exists and how it works.